src/docs/blueprint_documentation.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Packaging Theia as a Desktop Product
+---
+
+# Packaging Theia as a Desktop Product
+
+Eclipse Theia Blueprint product is a template to showcase the capabilities of Theia as well as how to build desktop-based products based on the platform.
+Theia Blueprint assembles a selected subset of existing Theia features and extensions.
+We provide installers for Theia Blueprint to be downloaded (see below) from the Theia webpage.
+In the respective git Repository you can also find the source code the product and its installers were built from.
+This is also the starting point for this documentation that is about how to customize this template to build your own custom Theia-based product and installers and packaging for installing the Desktop-based version of your custom product on all major operating systems.
```

Comment:
> This is also the starting point for this documentation that is about

This sentence does not read particularly well and can be reworded.

Reply:
Suggestion:
This documentation will use these sources as a template. We will explain how to customize this template so you can build your own custom Theia-based product including installers and packaging for installing the desktop-based version of your custom product on all major operating systems.